packs/chef/actions/lib/shellhelpers.py

```diff
@@ -43,17 +42,18 @@ def __init__(self, parser_options):
             self.parser.add_argument(k, lg, **kwargs)
             self._keyname[lg.lstrip('-')] = k
 
+
     def parse(self, argv=None):
-        argv = argv or sys.argv
```

Comment:
Yeah but actually its parse_args behavior is implicit:) Which for `args=None` takes `sys.argv[1:0]` and if it's given it takes the whole list. So your changes will break my code:
```
       argv = argv or sys.argv
       return vars(self.parser.parse_args(args=argv)) # since I expect args without the first one this won't work
```